app/mailers/user_mailer.rb

```diff
@@ -377,6 +377,18 @@ def admin_deleted_work_notification(user, work)
     end
   end
 
+  def admin_hidden_series_notification(creation_id, user_id)
```

Comment:
do we want to reutilize the `admin_hidden_work_notification` action, and maybe call it `admin_hidden_creation_notification`?

and update the email contents by passing in parameters?

Reply:
Personally, I like separate actions for each. I think the contents of the emails could potentially have enough differences that it could get kind of messy trying to account for all of the variations.

Reply:
Thank you! 

Added a separate action for bookmarks